### DISCUSSION TITLE
Time-lapse tutorial

### DISCUSSION THREAD

**Discussion by mariosgeo** [gimli-org/pyGIMLi#386] [Q&A]
Hi,

I am searching on the bert tutorial, where you mention that 

"From an inversion point of view, the most rigorous method is a full discretization in time and simultaneous inversion. This is done efficiently using block matrices but up to now available only in the Python managers"

I suppose this is the real 4D inversion as show by Kim paper. yet, i can't seem to find a tutorial how to do that. In my case, I have 50 .ohm files (one file per dataset). How would I use the time-lapse version?

Is there a tutorial you can point to me?

thanks in advance

**Comment by halbmy** [comment]
Dear Marios, there is no tutorial yet, but it is highly needed. Actually a simple task for me, I will set up such a framework.

**Comment by mariosgeo** [comment]
Fantastic Thomas.

Thanks a lot.

**Reply by mariosgeo** [reply]
Hi Thomas,

Did you had a chance to work on the tutorial?

**Comment by mariosgeo** [comment]
Revving this needed topic :) . Any tutorial on the horizon?

**Comment by halbmy** [comment]
Yes, it is already there, just not yet on the website as we need to fix things for the upcoming versions. But you can look here:
https://dev.pygimli.org/_examples_auto/3_ert/plot_09_timelapse.html

There is also a repository dedicated to timelapse ERT: https://github.com/gimli-org/timelapseERT

**Comment by mariosgeo** [comment]
Thanks Thomas. I will "play" with it.

**Comment by halbmy** [comment]
With the newest version v1.4.5 this is available "officially".
